src/openzaak/components/documenten/models.py

```diff
@@ -466,6 +468,20 @@ def locked(self, value: bool) -> None:
         self._locked = value
 
     def save(self, *args, **kwargs) -> None:
+        is_new_instance = self.pk is None
+        if is_new_instance and hasattr(self, "bronorganisatie"):
```

Comment:
I think `hasattr(self, "bronorganisatie")` will always evaluate to true, so I think that condition can be removed